tpchgen/src/distribution.rs

```diff
@@ -14,7 +14,12 @@ pub struct Distribution {
     name: &'static str,
     values: Vec<&'static str>,
     weights: Vec<i32>,
-    distribution: Option<Vec<&'static str>>,
+    /// Weighted text values generated from the distribution.
+    ///
+    /// If the table this distribution is for isn't actually a valid
+    /// distribution (e.g. 'nation'), then this vec will be empty, and
+    /// `max_weight` set to -1.
+    distribution: Vec<&'static str>,
```

Comment:
I removed the `Option` here since it was only telling us if this distribution is a valid distribution or not, but we can already glean that information from the `max_weight` being -1.

This removes the `Option` check in `random_value`.